src/writeData/subscriptions/components/JsonPathInput.tsx

```diff
@@ -48,7 +48,11 @@ const JsonPathInput: FC<Props> = ({
   itemNum,
   edit,
 }) => {
-  const dataTypeList = ['String', 'Number']
+  const stringType = 'String'
+  const floatType = 'Float'
+  const intType = 'Integer'
+  const booleanType = 'Boolean'
```

Comment:
Could we put these in `utils` so we only declare them once? Defining them here means that each time the component is re-rendered they are redefined which uses more browser memory